stable/insights-agent/templates/polaris/configmap.yaml

```diff
@@ -8,56 +8,25 @@ metadata:
     app: insights-agent
 data:
   config.yaml:  |
```

Comment:
I'd want to double check, but I think we can remove `config.yaml` now so that we just use the version that comes with Polaris

We should also cross-check the exemptions we have here with what's in Polaris...they fell out of sync at some point